doc/authentication-methods/client-certificate.md

```diff
@@ -0,0 +1,117 @@
+## Overview
+
+Clients connected to MongooseIM may authenticate with their TLS certificates.
+This method is a combination of `SASL EXTERNAL` mechanism and compatible certificate-aware backend.
```

Comment:
+This method is a combination of the `SASL EXTERNAL` mechanism and a compatible certificate-aware backend.